HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -3611,24 +3644,7 @@ def self.calc_duct_leakages_cfm25(distribution_system, system_cfm)
     return cfms[HPXML::DuctTypeSupply], cfms[HPXML::DuctTypeReturn]
   end
 
-  # This method is being deleted in https://github.com/NREL/OpenStudio-HPXML/pull/1707
-  def self.process_curve_fit(airflow_rate, capacity, temp)
-    # TODO: Get rid of this curve by using ADP/BF calculations
-    return 0 if capacity == 0
-
-    capacity_tons = UnitConversions.convert(capacity, 'Btu/hr', 'ton')
-    return MathTools.biquadratic(airflow_rate / capacity_tons, temp, get_shr_biquadratic)
-  end
-
-  # This method is being deleted in https://github.com/NREL/OpenStudio-HPXML/pull/1707
-  def self.get_shr_biquadratic
-    # Based on EnergyPlus's model for calculating SHR at off-rated conditions. This curve fit
-    # avoids the iterations in the actual model. It does not account for altitude or variations
-    # in the SHRRated. It is a function of ODB (MJ design temp) and CFM/Ton (from MJ)
-    return [1.08464364, 0.002096954, 0, -0.005766327, 0, -0.000011147]
-  end
-
-  # Determines the nominal speed (of a multi/variable-speed system).
+  # Determines the speed (of a multi/variable-speed system) that is running during the design conditions.
```

Comment:
Looks like you reverted a change here too. Please fix.

Reply:
@shorowit I made some local corrections to the comments above self.get_nominal_speed, committed them, and performed a git diff between the remote branch adp_bf_method_design_shr and remote master. In VS code terminal this is 

git diff adp_bf_method_design_shr origin/master  

There is no diff for the get_nominal_speed method now. What is in adp_bf branch matches master exactly.